ODIN_II/SRC/read_blif.cpp

```diff
@@ -399,6 +399,11 @@ void create_latch_node_and_driver(FILE *file, Hashtable *output_nets_hash)
 	output_nets_hash->add(new_node->name, new_net);
 
 	/* Free the char** names */
+	for (i = 0; i < input_token_count; i++)
+		// Except these two since they were assigned to new_node
+		if (i!=0 && i!=3)
+			vtr::free(names[i]);
+
```

Comment:
do strdup at line 371 and line 376 rather and here simply free them all
avoid complicated logic like this at all cost, change may occur in the code base and this would produce some unexpected side effect down the line